test.env.example

```diff
@@ -1,3 +1,17 @@
-BIGQUERY_TEST_DATABASE=
-BIGQUERY_TEST_ALT_DATABASE=
-BIGQUERY_TEST_SERVICE_ACCOUNT_JSON='{}'
\ No newline at end of file
+# Note: These values will come from your Bigquery account and gcp projects. 
+If you don't have an account set up yet take note of the fields below and gather them as you create one.
+
+### Test Environment field definitions 
+# These will all be gathered from account information or created by you.
+BIGQUERY_TEST_DATABASE: Name of the gcp project your project is using
+BIGQUERY_TEST_NO_ACCESS_DATABASE: Name of a gcp project you don't have access to query.
+BIGQUERY_TEST_ALT_DATABASE: Name of a second gcp project on your account.
+BIGQUERY_TEST_SERVICE_ACCOUNT_JSON: Service account JSON key created in your Service Accounts page on bigquery console
+
+# Copy the following to a test.env, and replace example values with your information.
+BIGQUERY_TEST_DATABASE=name-of-my-project
+BIGQUERY_TEST_NO_ACCESS_DATABASE=name-of-someone-elses-project (please make sure to have permission from creator to reference in your local files)
+BIGQUERY_TEST_ALT_DATABASE=name-of-another-project-of-one-of-my-projects
```

Comment:
Oh man I'm having trouble parsing this or knowing what to suggest otherwise. hmmm....maybe clarify the word project? Name of alternate project in gcp cluster? I really don't know bigquery all that well to say for sure.